lib/hyrax/rails/routes.rb

```diff
@@ -46,20 +46,6 @@ def curation_concerns_basic_routes(&block)
       end
     end
 
-    # kmr added :show to make tests pass
-    def curation_concerns_embargo_management
```

Comment:
keep the method, make it do nothing and have:

```ruby
Deprecation.warn(self, "curation_concerns_embargo_management is deprecated and has no effect. It will be removed in Hyrax 3.0")
```